admin_panel/src/commonMain/kotlin/net/thechance/mena/admin_panel/data/mapper/pagedResponseMapper.kt

```diff
@@ -0,0 +1,8 @@
+package net.thechance.mena.admin_panel.data.mapper
+
+import net.thechance.mena.admin_panel.data.remote.dto.PagedResponse
+import kotlin.collections.orEmpty
+
+fun <DTO, ENTITY> PagedResponse<DTO>.toEntityList(mapper: DTO.() -> ENTITY): List<ENTITY> {
+    return this.items?.map { it.mapper() }.orEmpty()
+}
```

Comment:
what is your opinion about this change ?

```suggestion
fun <DTO, ENTITY> PagedResponse<DTO>.toEntityList(mapper: DTO.() -> ENTITY): List<ENTITY> =
    this.items?.map { it.mapper() }.orEmpty()
```